File: fizzbuzz.kt
fun main() {
	println("***** Hello kotlin! *****")
	
	for (num in 1..35) {
        print(
            when {
                num % 15 == 0 -> "FizzBuzz"
                num % 3 == 0 -> "Fizz"
                num % 5 == 0 -> "Buzz"
                else -> num
            }
        )
        print(" ")
    }

}

File: FizzBuzz.py
def fizzbuzz(max):
    for i in range(1, max+1) :
        msg = "fizz" * (i % 3 == 0) + "buzz" * (i % 5 == 0)
        print((msg if msg else i), end=' ')
    print('')

fizzbuzz(100)